### ISSUE TITLE
Single input, Single output

### ISSUE THREAD

**Issue by Shroototem** [BrainJS/brain.js#687]
## Issue
 I'm unable to use a single input to single output training.

## _How_ do we replicate the issue?
```js
const brain = require('brain.js')
const fs = require('fs')

const net = new brain.NeuralNetwork()

var TraningData = [
    { input: [1], output: [1]},
    { input: [2], output: [4]},
    { input: [3], output: [6]},
    { input: [4], output: [8]},
    { input: [5], output: [10]}
  ]

net.train(TraningData, {
    errorThresh: 0.001,
    iterations: 100000,
    callback: step => console.log(step)
})
```

The 'errorThresh' never goes down.

## Version information
brain.js@2.0.0-beta.2

### Nodejs:
v14.15.3

## _How_ important is this (1-5)?
1

**Comment by Shroototem** [comment]
```js
const brain = require('brain.js')
const fs = require('fs')

const net = new brain.recurrent.RNNTimeStep({
  inputSize: 1,
  outputSize: 1
})
net.train(
  [
    { input: [2], output: [4] },
    { input: [3], output: [6] },
    { input: [4], output: [8] },
    { input: [10], output: [20] },
  ],
  {
    errorThresh: 0.00001,
    iterations: 10000000,
    callback: step => console.log(step)
  })
  ```
  
  No matter what I put in, it gives me the wrong output.
  If you input 10, it outputs 0.1329
  Of course this is supposed to be 20.

**Comment by robertleeplummerjr** [comment]
Don't use a time step or rnn for a single input and single output.  There is no recurrence.  Use the standard neural network.    I'll get you a working example shortly.

**Comment by robertleeplummerjr** [comment]
I think the main thing here is that you need to normalize and denormalize the data.  Full working example: https://observablehq.com/@robertleeplummerjr/brain-js-issue-687

**Comment by Shroototem** [comment]
Why would I have to go in such a round about way for this? Is this not able to just multiply the numbers by 2?

**Comment by Shroototem** [comment]
How do I call this later on?
```js
const brain = require('brain.js')
const fs = require('fs')

const json = JSON.parse(fs.readFileSync('trained-net.json'))
const net = new brain.NeuralNetwork()
net.fromJSON(json)

let rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
})
rl.question('Input?: ', answer => {
    console.log(net.run(answer))
    rl.close()
})
```
I'm getting a float from 0 to 1.
You also set the max to 20. What if I try to put in say 34, would it output 68?

**Comment by robertleeplummerjr** [comment]
The normalization functions I added in the example work dynamically, do changing the training data will still normalize accordingly.  You'll get values between 0 and 1 until you denormalize them to original values, which is in the example as well.

```ts
return (
    []
      .concat(normalizedTrainingData.map(datum => net.run(datum.input)))
      .concat(normalizedTrainingData.map(datum => newNet.run(datum.input)))
    )
    .map(output => denormalizeData(bounds, output));
```

**Comment by robertleeplummerjr** [comment]
Updated example to be dynamic in showing results as well.

**Comment by Shroototem** [comment]
Sorry to bother you but I think I'm going to look elsewhere for what I need.
